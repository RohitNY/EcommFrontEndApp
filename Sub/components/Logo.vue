<template>
  <img :class="$style.logo" src="~/assets/images/pch-logo.png">
</template>

<script>
export default {

}
</script>

<style module>
.logo {
  width: 180px;
}
</style>
