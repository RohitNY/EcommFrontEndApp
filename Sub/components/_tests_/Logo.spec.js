import { shallowMount } from '@vue/test-utils'
import Logo from '~/components/Logo.vue'

describe('Logo', () => {
  const wrapper = shallowMount(Logo)

  test('is a Vue instance', () => {
    expect(wrapper.findComponent(Logo)).toBeTruthy()
  })

  // test('matches snapshot', () => {
  //   expect(wrapper.html()).toMatchSnapshot()
  // })
})
