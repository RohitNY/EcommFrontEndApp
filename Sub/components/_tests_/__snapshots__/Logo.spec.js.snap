// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo matches snapshot 1`] = `"<img src=\\"~/assets/images/pch-logo.png\\" class=\\"logo\\">"`;
